libioc/Jail.py

```diff
@@ -1940,6 +1938,31 @@ def _start_vimage_network(self) -> typing.Tuple[
 
         return created, start
 
+    def _stop_network(self) -> typing.List[str]:
+        if self.config["vnet"]:
+            return self._stop_vimage_network()
+        else:
+            return self._stop_non_vimage_network()
+
+    def _stop_non_vimage_network(self) -> typing.List[str]:
+        commands: typing.List[str] = []
+        for protocol in (4, 6,):
+            config_value = self.config[f"ip{protocol}_addr"]
+            if config_value is None:
+                return
+            for nic, addresses in config_value.items():
+                if addresses is None:
+                    continue
+                for address in addresses:
+                    if isinstance(address, str):
+                        # skip DHCP and ACCEPT_RTADV
+                        continue
```

Comment:
why?

Reply:
They are not accepted by jail during start. So it must be a mistake that should not prevent the teardown.